pubspec.yaml

```diff
@@ -1,54 +1,21 @@
 name: permission_handler
 description: Permission plugin for Flutter. This plugin provides a cross-platform (iOS, Android) API to request and check permissions.
-version: 2.2.0
-author: Baseflow <hello@baseflow.com>
+version: 3.0.0
+authors:
+  - Baseflow <hello@baseflow.com>
+  - long1eu <home@long1.eu>
 homepage: https://github.com/baseflowit/flutter-permission-handler
 
+environment:
+  sdk: ">=2.1.0 <3.0.0"
```

Comment:
Any idea why this is causing the build on Travis-CI to fail?

```
The current Dart SDK version is 2.1.0-dev.9.4.flutter-f9ebf21297.
Because permission_handler_example requires SDK version >=2.1.0 <3.0.0, version solving failed.
---- Log transcript ----
FINE: Pub 2.1.0-dev.9.4.flutter-f9ebf21297
MSG : Resolving dependencies...
SLVR: fact: permission_handler_example is 0.0.0
SLVR: derived: permission_handler_example
SLVR: fact: permission_handler_example requires SDK version >=2.1.0 <3.0.0
SLVR: conflict: permission_handler_example requires SDK version >=2.1.0 <3.0.0
SLVR: Version solving took 0:00:00.030740 seconds.
    | Tried 1 solutions.
FINE: Resolving dependencies finished (0.0s).
ERR : The current Dart SDK version is 2.1.0-dev.9.4.flutter-f9ebf21297.
    | 
    | Because permission_handler_example requires SDK version >=2.1.0 <3.0.0, version solving failed.
FINE: Exception type: SolveFailure
FINE: package:pub/src/solver/version_solver.dart 308:5   VersionSolver._resolveConflict
    | package:pub/src/solver/version_solver.dart 131:27  VersionSolver._propagate
    | package:pub/src/solver/version_solver.dart 96:9    VersionSolver.solve
    | ===== asynchronous gap ===========================
    | dart:async                                         _AsyncAwaitCompleter.completeError
    | package:pub/src/solver/version_solver.dart         VersionSolver.solve
    | ===== asynchronous gap ===========================
    | dart:async                                         _asyncThenWrapperHelper
    | package:pub/src/solver/version_solver.dart         VersionSolver.solve
    | package:pub/src/solver.dart 35:10                  resolveVersions.<fn>
    | package:pub/src/log.dart 378:18                    progress
    | package:pub/src/solver.dart 32:10                  resolveVersions
    | package:pub/src/entrypoint.dart 200:24             Entrypoint.acquireDependencies
---- End log transcript ----
pub get failed (1)
```

I think we could update this to:
```suggestion
  sdk: ">=2.0.0 <3.0.0"
```

Reply:
Trevis is using the stable branch. I missed that. :D